cogs5e/models/initiative.py

```diff
@@ -688,6 +688,23 @@ def group(self):
     def group(self, value):
         self._group = value
 
+    def set_group(self, group_name: str):
+        current = self.combat.current_combatant
+        was_current = self is current or \
+                      (isinstance(current, CombatantGroup) and self in current and len(current) == 1)
+        self.combat.remove_combatant(self, ignore_remove_hook=True)
+        if group_name.lower() == 'none':
```

Comment:
While we're here, may as well make this accept a `None` group name to clear groups rather than the string `'none'`